corehq/apps/translations/integrations/transifex/views.py

```diff
@@ -326,11 +329,24 @@ def page_context(self):
             context['backup_form'] = AppTranslationsForm.form_for('backup')(self.domain)
             if self.request.user.is_staff:
                 context['delete_form'] = AppTranslationsForm.form_for('delete')(self.domain)
+            context['blacklisted_translations'] = self._blacklisted_translations
+            context['blacklist_form'] = AddTransifexBlacklistForm(self.domain)
         form_action = self.request.POST.get('action')
         if form_action:
             context[form_action + '_form'] = self.translations_form
         return context
 
+    @property
+    def _blacklisted_translations(self):
+        blacklisted = TransifexBlacklist.objects.filter(domain=self.domain).all().values()
+        app_ids_to_name = {app.id: app.name for app in get_brief_apps_in_domain(self.domain)}
+        ret = []
+        for trans in blacklisted:
+            r = trans.copy()
+            r['app_name'] = app_ids_to_name.get(trans['app_id'], trans['app_id'])
```

Comment:
I tried looking to see if I could easily get the module name instead of id, but I think that would require pulling all the apps that are reference to find that. There's no easy view to get that info.

Reply:
makes sense to me